test/e2e/suites/managed/machine_deployment_helpers.go

```diff
@@ -58,7 +58,7 @@ func waitForMachineDeploymentDeleted(ctx context.Context, input waitForMachineDe
 		err := input.Getter.Get(ctx, key, mp)
 		notFound := apierrors.IsNotFound(err)
 		return notFound
-	}, intervals...).Should(BeTrue())
+	}, intervals...).Should(BeTrue(), fmt.Sprintf("Eventually failed waiting for MachineDeployment %s to be deleted", input.MachineDeployment.GetName()))
```

Comment:
Can you please change _all occurrences_ into `%q` instead of `%s`? If a value is empty by mistake, which is very likely when writing tests, it's super-hard to see two spaces as compared to a quoted string that clearly says it's empty: `waiting for MachineDeployment  to be deleted` vs. `waiting for MachineDeployment "" to be deleted`.

Reply:
Good suggestion! Updated thanks Andreas!